env/staging/ecs/task-definitions/metrics.json

```diff
@@ -37,6 +37,12 @@
           "value": "${bucket_name}"
         }
       ],
+      "containerDefinitions": [{
+        "secrets": [{
+          "name": "metrics_token",
+          "valueFrom": "${metric_token_secret_arn}"
+        }]
+      }],
```

Comment:
We should make a separate task definition for the server metrics task since the other two types of metric tasks don't need to know this secret.